VDA-DID-Registry/contracts/VeridaDIDRegistryV2.sol

```diff
@@ -1,232 +1,11 @@
-    _registeredDIDs.remove(didAddress);
-
-    emit Revoke(didAddress);
-  }
-
-  /**
-   * @notice Internal function to get a controller of DID address
-   * @dev This is internal function to be used in another functions. Gas efficient than external function
-   * @param didAddress DID address
-   * @return result Controller of DID address
-   */
-  function _getController(address didAddress) internal view returns(address result) {
-    result = _DIDInfo[didAddress].controller;
-    assembly {
-      if iszero(result) {
-        result := didAddress
-      }
-    }
-  }
-
-  /**
-   * @dev See {IVeridaDIDRegistry}
-   */
-  function getController(address didAddress) external view override returns(address) {
-    return _getController(didAddress);
-  }
-
-  /**
-   * @dev See {IVeridaDIDRegistry}
-   */
-  function setController(address didAddress, address controller, bytes calldata signature) external override {
-    if (!isRegistered(didAddress)) {
-      revert UnregisteredDID();
-    }
-    {
-      address oldController = _getController(didAddress);
-      bytes memory rawMsg = abi.encodePacked(
-        didAddress, 
-        "/setController/",
-        controller,
-        "/",
-        _nonce[didAddress]);
-      
-      if (!VeridaDataVerificationLib.validateSignature(rawMsg, signature, oldController)) {
-        revert InvalidSignature();
-      }
-      _nonce[didAddress]++;
-    }
-
-    _DIDInfo[didAddress].controller = controller;
-
-    emit SetController(didAddress, controller);
-  }
-
-  /**
-   * @dev See {IVeridaDIDRegistry}
-   */
-  function lookup(address didAddress) external view override returns(address, string[] memory) {
-    if (!isRegistered(didAddress)) {
-      revert UnregisteredDID();
-    }
-
-    EnumerableSet.StringSet storage list = _DIDInfo[didAddress].endpoints;
-    uint length = list.length();
-
-    string[] memory ret = new string[](length);
-
-    for (uint i; i < length;) {
-      ret[i] = list.at(i);
-      unchecked { ++i; }
-    }
-
-    return (_getController(didAddress), ret);
-  }
-
-  /**
-   * @dev See {IVeridaDIDRegistry}
-   */
-  function nonce(address didAddress) external view override returns(uint) {
-    return _nonce[didAddress];
-  }
-
-  /**
-   * @dev See {IVeridaDIDRegistry}
-   */
-  function activeDIDCount() external view override returns(uint) {
-    return _registeredDIDs.length();
-  }
-
-  /**
-   * @dev See {IVeridaDIDRegistry}
-   */
-  function getDIDs(uint startIndex, uint count) external view override returns(address[] memory) {
-    if (count == 0 || (startIndex + count ) > _registeredDIDs.length()) {
-      revert OutOfRange();
-    }
-
-    address[] memory ret = new address[](count);
-
-    for (uint i; i < count;) {
-      ret[i] = _registeredDIDs.at(startIndex + i);
-      unchecked { ++i; }
-    }
-
-    return ret;
+contract VeridaDIDRegistryV2 is VeridaDIDRegistry {
+  function getVerison() external pure virtual returns(string memory) {
```

Comment:
Typo

Reply:
Add a unit test for this.

Reply:
This `VeridaDIDRegistryV2` contract was created for upgrade test in the testnet.
What unit tests do you need?

Reply:
Just a test that `getVersion()` returns a value.

That would pickup the typo (`getVerison()`)

Reply:
Updated